### DISCUSSION TITLE
Query on customization of report

### DISCUSSION THREAD

**Discussion by yschan-ntu** [aiverify-foundation/aiverify#87] [Q&A] [answered]
Hi, I've been playing around with AIVerify on a windows machine with Docker.
 
A quick question, I know that currently it is possible to customize the sections within the report in the GUI (specifically within the sections within the process checklist), but is it possible to customize the input questions within the sections in the process checklist in the GUI?

Thanks much!

**Comment by kimeetok** [comment, marked as answer]
Hi @yschan-ntu thanks for trying out out toolkit! The input questions within each process checklist cannot be customised as they are a set of required checks defined by the AI Verify Testing Framework. If you wish to create your own checklist, you can explore building an input block plugin for that using our [plugin development tool](https://imda-btg.github.io/aiverify-developer-tools/plugins/widget/Plugin_Tool/) :)

**Reply by yschan-ntu** [reply]
Noted with thanks!
